### DISCUSSION TITLE
怎么安装啊

### DISCUSSION THREAD

**Discussion by ssj952** [ChatGPTNextWeb/NextChat#1181] [Q&A] [answered]
这个东西怎么安装啊，作者好像没有给攻略，我看作者下面的介绍又不是安装介绍

**Comment by CKylinMC** [comment, marked as answer]
https://github.com/Yidadaa/ChatGPT-Next-Web/blob/main/README_CN.md

Readme的开始使用一节和部署一节，一个是用免费资源，一个是自己部署。

如果发现这些都不是你想要的，那么你可能需要的是现成的东西，不妨去看看别家收集的免费或付费的gpt网站或者客户端的项目。
